tests/python/contrib/test_hexagon/benchmark_elemwise_add.py

```diff
@@ -0,0 +1,468 @@
+    """
+    Similar to `test_elemwise_add_te`, but starting with TensorScript rather than
+    Tensor Expressions.
+    """
+
+    # Create and benchmark a single primfunc.
+    # If an unexpected problem occurs, raise an exception.  Otherwise add a row of output to 'bt'.
+    def test_one_config(dtype, mem_scope, num_vectors_per_tensor):
+        basic_kernel = "ewise-tvmscript-1"
+
+        # The distinguishing characteristics of this benchmark line item.
+        keys_dict = {
+            "basic_kernel": basic_kernel,
+            "dtype": dtype,
+            "mem_scope": mem_scope,
+            "num_vectors_per_tensor": num_vectors_per_tensor,
+        }
+
+        host_files_dir_name = _get_benchmark_id(keys_dict)
+        desc = _get_benchmark_decription(keys_dict)
+
+        print(f"CONFIGURATION: {desc}")
+
+        if num_vectors_per_tensor == 2048 and mem_scope == "global.vtcm":
+            _BT.record_skip(**keys_dict, comments="Expect to exceed VTCM budget.")
+            return
+
+        host_files_dir = os.path.join(_HOST_OUTPUT_DIR, host_files_dir_name)
+        os.mkdir(host_files_dir)
+
+        dtype_bits = tvm._ffi.runtime_ctypes.DataType(dtype).bits
+        assert dtype_bits % 8 == 0
+        dtype_bytes = dtype_bits // 8
+
+        elem_per_hvx_vector = HVX_VECTOR_BYTES // dtype_bytes
+
+        shape = [
+            num_vectors_per_tensor,
+            elem_per_hvx_vector,
+        ]
+
+        # TVMScript can reference simple Python variables, but it doesn't
+        # curently support more complex Python expressions...
+        dim0_size = shape[0]
+        dim1_size = shape[1]
+        dtype_str = str(dtype)
+
+        @tvm.script.ir_module
+        class MyModule:
+            @T.prim_func
+            def main(a: T.handle, b: T.handle, c: T.handle):
+                # We exchange data between function by handles, which are similar to pointer.
+                T.func_attr({"global_symbol": "main", "tir.noalias": True})
+                # Create buffer from handles.
+                A = T.match_buffer(a, shape, dtype=dtype_str)
+                B = T.match_buffer(b, shape, dtype=dtype_str)
+                C = T.match_buffer(c, shape, dtype=dtype_str)
+
+                for i in range(dim0_size):
+                    for j in range(dim1_size):
+                        C[i, j] = A[i, j] + B[i, j]
+
+        ir_module = MyModule
+
+        A = tvm.te.placeholder(shape, dtype=dtype)
+        B = tvm.te.placeholder(shape, dtype=dtype)
+        C = tvm.te.placeholder(shape, dtype=dtype)
+
+        module_for_ir_dump = tvm.lower(ir_module, [A, B, C], "elemwise_add")
+
+        report_path = os.path.join(host_files_dir, "out.txt")
+        with open(report_path, "w") as f:
+            f.write("LOWERED IR MODULE:\n")
+            f.write(str(module_for_ir_dump))
+            f.write("\n")
+
+            target_hexagon = tvm.target.hexagon("v68", link_params=True)
+            func = tvm.build(
+                # sched,
```

Comment:
This should be removed.  It can be re-added once we're actually using a schedule.